optuna/multi_objective/_hypervolume/wfg.py

```diff
@@ -15,14 +15,14 @@ class WFG(BaseHypervolume):
     """
 
     def __init__(self) -> None:
-        self._reference_point = None
```

Comment:
I think it's clearer to type hint this member as an `Optional[np.array]` instead of changing the logic (and add `assert self._reference_point is not None` where needed).